src/battle_debug.c

```diff
@@ -800,8 +800,15 @@ static void Task_ShowAiPoints(u8 taskId)
                 data->spriteIds.aiIconSpriteIds[i] = 0xFF;
             }
         }
+
+        struct Pokemon *mon;
+        if (GetBattlerSide(data->aiBattlerId) == B_SIDE_PLAYER)
+            mon = &gPlayerParty[gBattlerPartyIndexes[data->aiBattlerId]];
+        else
+            mon = &gEnemyParty[gBattlerPartyIndexes[data->aiBattlerId]];
```

Comment:
I think there is a function / macro to get a side mon.

Reply:
You're right! `GetBattlerParty`